src/dbg/value.cpp

```diff
@@ -1702,6 +1723,8 @@ bool valfromstring_noexpr(const char* string, duint* value, bool silent, bool ba
             *isvar = true;
         return true;
     }
+    else if(valenvfromstring(string, value)) //environment block

```

Comment:
Hm, probably I'll move this to `exprfuncs.cpp` so you can use `teb()` and `peb()`... Doesn't hurt to have though.